cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -238,24 +243,21 @@ class EvaluatorCommand : OrtCommand(
             }
         }
 
-        val script = when (rules) {
-            is FileType -> (rules as FileType).file.readText()
+        if (checkSyntax) {
+            val evaluator = Evaluator()
 
-            is StringType -> {
-                val rulesResource = (rules as StringType).string
-                javaClass.getResource(rulesResource)?.readText()
-                    ?: throw UsageError("Invalid rules resource '$rulesResource'.")
-            }
-        }
+            var allChecksSucceeded = true
 
-        if (checkSyntax) {
-            if (Evaluator().checkSyntax(script)) {
-                println("Syntax check succeeded.")
-                return
+            scriptUrls.forEach {
+                if (evaluator.checkSyntax(it.readText())) {
+                    println("Syntax check for $it succeeded.")
```

Comment:
Maybe single quotes around `$it`? (same below)

Reply:
These are `URL`s which we usually don't quote.